packages/storybook/stories/va-radio.stories.jsx

```diff
@@ -13,19 +13,30 @@ export default {
   subcomponents: componentStructure(radioItem),
 };
 
-const Template = args => {
+const Template = ({
+  'enable-analytics': enableAnalytics,
```

Comment:
I'm not sure if renaming the prop here is going to do what we want.

Reply:
The latest changes addresses this. The web component helper propStructure is now using the prop attr instead of prop name.